crowbar_framework/db/migrate/20150312081459_create_proposals.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProposals < ActiveRecord::Migration
+  def change
+    create_table :proposals do |t|
+      t.string :barclamp, :null => false
+      t.string :name, :null => false
+      t.text :properties
```

Comment:
@bkutil so it's a hunch, but based on several things:

* most bits in the deployment part of the proposal should actually never be exposed to the user (nor should the user be allowed to change them, imho)
* the revisions and crowbar-applied bits that are in the deployment part should likely just be straight columns in the database anyway
* having the role assignments available in a different table and in an easy way to query is something that, I guess, will be useful longer term (for the orchestration)
* the other bits should really be static

That being said, I'd definitely agree with doing that as a later step; so a research card is fine with me.